oar/core/worksheet_mgr.py

```diff
@@ -0,0 +1,311 @@
+        Update task status in check list
+        e.g. Pass/Fail/In Progress
+
+        Args:
+            label (str): cell label, A1/B2
+            status (str): Pass/Fail/In Progress
+        """
+        self._ws.update_acell(label, status)
+
+    def get_task_status(self, label):
+        """
+        Get task status
+
+        Args:
+            label (str): cell label of different tasks
+        """
+        return self._ws.acell(label).value
+
+    def is_task_in_progress(self, label):
+        """
+        Util func to check whether task is working in progress
+
+        Args:
+            label (str): cell label of different tasks
+        """
+        return TASK_STATUS_INPROGRESS == self.get_task_status(label)
+
+    def is_task_pass(self, label):
+        """
+        Util func to check whether task is pass
+
+        Args:
+            label (str): cell label of different tasks
+        """
+        return TASK_STATUS_PASS == self.get_task_status(label)
+
+    def is_task_fail(self, label):
+        """
+        Util func to check whether task is fail
+
+        Args:
+            label (str): cell label of different tasks
+        """
+        return TASK_STATUS_FAIL == self.get_task_status(label)
+
+    def is_task_not_started(self, label):
+        """
+        Util func to check whether task is not started
+
+        Args:
+            label (str): cell label of different tasks
+        """
+        return TASK_STATUS_NOT_STARTED == self.get_task_status(label)
+
+    def generate_bug_list(self, jira_issues: []):
+        """
+        Generate bug list of on_qa bugs
+
+        Args:
+            jira_issues (str[]): jira issue keys from advisories
+        """
+        jm = JiraManager(self._cs)
+        row_idx = 8
+        batch_vals = []
+        for key in jira_issues:
+            issue = jm.get_issue(key)
+            logger.debug(f"updating jira issue {key} ...")
+            if issue.is_on_qa():
+                logger.debug(f"jira issue {key} is ON_QA, updating")
+                row_vals = []
+                row_vals.append(self._to_hyperlink(self._to_jira_link(key), key))
+                row_vals.append(issue.get_qa_contact())
+                row_vals.append(issue.get_status())
+                batch_vals.append(row_vals)
+                row_idx += 1
+            else:
+                logger.debug(
+                    f"jira issue {key} status is {issue.get_status()}, skipping"
+                )
+        self._ws.batch_update(
+            [
+                {
+                    "range": LABEL_BUG_FIRST_CELL + ":E" + str(row_idx),
+                    "values": batch_vals,
+                }
+            ],
+            value_input_option=gspread.utils.ValueInputOption.user_entered,
+        )
+        # TODO: highlight the cell if the issue is critical
+        # TODO: send email/slack notification
+
+        logger.info("bugs to be verified are updated")
+
+    def update_bug_list(self, jira_issues: []):
```

Comment:
@rioliu-rh will it be done? reminder: incremental update.

Reply:
yes, this func will update bug list with existing cell data